src/utils/share/index.js

```diff
@@ -42,7 +42,7 @@ const sharePostInTopic = async (item, analyticsLogEvent, analyticsId) => {
 };
 
 const shareCommunity = (topicname) => {
-  buildShare(`${config.POST_SHARE_URL}/community/${topicname}`);
+  buildShare(`${config.POST_SHARE_URL}/c/${topicname}`);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The URL structure has been changed from `/community/` to `/c/`. Ensure that this change is reflected in the backend and all other places where this URL is used. If not, it might lead to broken links.
```diff
-  buildShare(`${config.POST_SHARE_URL}/community/${topicname}`);
+  buildShare(`${config.POST_SHARE_URL}/c/${topicname}`);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->